_includes/_header.htm

```diff
@@ -4,7 +4,7 @@
             <div class="mx-auto">
                 <a href="#main" class="sr-only sr-only-focusable bg-dark text-light">Jump to Content</a>
                 <a name="Nav"></a> <!-- Nav link, returning user to top of website -->
-                <a class="navbar-brand" href="/" rel="tooltip" data-toggle="tooltip" title="Gridcoin - Securely rewarding BOINC computation on top of POS!" data-placement="bottom"><img src="/assets/img/grc-title2.png" alt="Gridcoin title" /></a>
+                <a class="navbar-brand" href="/" rel="tooltip" data-toggle="tooltip" title="Gridcoin - Securely rewarding BOINC computation on top of POS!" data-placement="bottom"><img src="/assets/img/grc-title.svg" alt="Gridcoin title" style="width: 250px"/></a>

```

Comment:
Not sure if this was intentional, but the width is set different (250px) from the current size (205px).